src/party_menu.c

```diff
@@ -6115,7 +6115,7 @@ void ChooseMonForWirelessMinigame(void)
 
 static u8 GetPartyLayoutFromBattleType(void)
 {
-    if (IsDoubleBattle() == FALSE)
+    if (!IsDoubleBattle() || gPlayerPartyCount == 1) // Draw the single layout in a double battle where the player has only one pokemon.
```

Comment:
I assume this should also be CountBattleReadyMons but it might need some testing